metadata_backend/api/handlers.py

```diff
@@ -366,6 +366,37 @@ async def patch_folder(self, req: Request) -> Response:
         LOG.info(f"PATCH folder with ID {folder} was successful.")
         return web.Response(body=body, status=200, content_type="application/json")
 
+    async def publish_folder(self, req: Request) -> Response:
+        """Update object folder specifically into published state.
+
+        :param req: PATCH request
```

Comment:
this is a `patch` but in routes it looks like a `.get` .... maybe we can settle on something